### PR TITLE
Minor updates before building rc1

### PR DESCRIPTION
**Pull request by fgiorgetti**

### CODE REVIEW

File: bundle/manifests/skupper-operator.v0.6.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -3,10 +3,21 @@ kind: ClusterServiceVersion
 metadata:
   annotations:
     alm-examples: '[]'
-    capabilities: Basic Install
+    certified: "false"
+    categories: "Integration & Delivery"
```
Comment (grs) [inline bundle/manifests/skupper-operator.v0.6.0.clusterserviceversion.yaml:7]: Possibly expand the categories? E.g. include Networking & Messaging? (What does 'Delivery' mean in this context?)

**Review by grs — APPROVED** [review]

File: bundle/manifests/skupper-operator.v0.6.0.clusterserviceversion.yaml (hunk 1 above)
Comment (fgiorgetti) [reply bundle/manifests/skupper-operator.v0.6.0.clusterserviceversion.yaml:7]: I have chosen this particular category based on the existing ones available at the Operator Hub.
But I believe we need to decide which one is the most appropriate to be used here.

Comment (ajssmith) [reply bundle/manifests/skupper-operator.v0.6.0.clusterserviceversion.yaml:7]: From interconnect-operator we had categories -> "Networking, Streaming & Messaging"

Suggest: "Integration, Networking, Streaming, Messaging"

**Review by ajssmith — APPROVED** [review]

File: bundle/manifests/skupper-operator.v0.6.0.clusterserviceversion.yaml (hunk 1 above)
Comment (fgiorgetti) [reply bundle/manifests/skupper-operator.v0.6.0.clusterserviceversion.yaml:7]: Here is the source for the allowed categories:
https://github.com/operator-framework/community-operators/blob/master/categories.json

These categories above can be defined as a comma separated list of strings.

Can we go with: "Integration & Delivery, Networking, Streaming & Messaging" ?

Comment (ajssmith) [reply bundle/manifests/skupper-operator.v0.6.0.clusterserviceversion.yaml:7]: +1

**Pull request merged**